### PR TITLE
New blog post about unified observability with opensearch

### PR DESCRIPTION
**Pull request by jkowall**
### Description
Adding new blog around unified observability 

_Explore how OpenSearch offers a unified observability solution for platform teams, addressing challenges of open-source tools by integrating with CNCF projects like Prometheus, Grafana, and Jaeger. Learn about the strategic advantages of OpenSearch in managing observability data, and get a preview of an in-depth demo at OpenSearchCon, showcasing its role in enhancing developer platforms._

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by jkowall** [comment]
Please process my blog, it's been in queue for 3 weeks and other blogs are going out. This is timely with the conference coming up. Thank you.

File: _posts/2024-08-07-OpenSearch-for-Unified-Observability.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Analysis using multiple interfaces
+
+OpenSearch is not only a database, but the powerful OpenSearch Dashboards allows the user to analyze observability data, namely logs and traces. In Jaeger (a project I am one of the maintainers of) we are working on some derived metrics on OpenSearch, but this is one weak spot for the database. I would not suggest doing high scale metrics on OpenSearch as It’s not suited for time series data. However, OpenSearch integrates with popular metric systems such as Prometheus, which is a better path forward for widespread use of metrics. 
+
+Although OpenSearch Dashboards is not as good as some other time series analysis tools such as Grafana, it doesn’t mean you can’t use both. Keep in mind that OpenSearch will never change licenses from its current Apache 2 license. This is not the case with other tools that are open source, but not part of software foundations such as the Apache Software Foundation, The Linux Foundation (LF) or the CNCF which is a sub-foundation of LF.
```
Comment (anirudha) [inline _posts/2024-08-07-OpenSearch-for-Unified-Observability.md:33]: seems like a subjective comparative statement, can you help baseline or quantify ?

**Review by anirudha — CHANGES_REQUESTED** [review]
added comment/ thanks

File: _posts/2024-08-07-OpenSearch-for-Unified-Observability.md (hunk 1 above)
Comment (jkowall) [reply _posts/2024-08-07-OpenSearch-for-Unified-Observability.md:33]: This is my opinion based on in depth market understanding I'm happy to provide a detailed write-up of what's missing but I don't think this is a blog to compare OpenSearch to commerical software.

**Comment by jkowall** [comment]
How can we get this published?

**Comment by krisfreedain** [comment]
@pajuric @natebower - do we know if this is going into editorial review soon? It need to be published for @jkowall before OpenSearchCon - thanks!

File: _posts/2024-08-07-OpenSearch-for-Unified-Observability.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+OpenSearch provides a path forward which can be strategic, especially as the project moves towards being part of a foundation. This allows the project to have closer collaboration and alignment with an ecosystem of observability projects supported by the Cloud Native Computing Foundation (CNCF). 
+
+## Analysis using multiple interfaces
+
+OpenSearch is not only a database, but the powerful OpenSearch Dashboards allows the user to analyze observability data, namely logs and traces. In Jaeger (a project I am one of the maintainers of) we are working on some derived metrics on OpenSearch, but this is one weak spot for the database. I would not suggest doing high scale metrics on OpenSearch as It’s not suited for time series data. However, OpenSearch integrates with popular metric systems such as Prometheus, which is a better path forward for widespread use of metrics.
```
Comment (brijos) [inline _posts/2024-08-07-OpenSearch-for-Unified-Observability.md:31]: Typo 

OpenSearch as It’s not suited for time series data 

should read

 OpenSearch as it’s not suited for time series data

**Comment by natebower** [comment]
> @pajuric @natebower - do we know if this is going into editorial review soon? It need to be published for @jkowall before OpenSearchCon - thanks!

@krisfreedain I haven't received a request for this one yet, but I'm happy to review it either tomorrow or (more likely) Monday.

File: _posts/2024-08-07-OpenSearch-for-Unified-Observability.md (hunk 2 above)
Comment (jkowall) [reply _posts/2024-08-07-OpenSearch-for-Unified-Observability.md:31]: Good catch, fixed.

**Comment by jkowall** [comment]
> @krisfreedain I haven't received a request for this one yet, but I'm happy to review it either tomorrow or (more likely) Monday.

How does one open a request?

**Comment by pajuric** [comment]
@brijos - Please advise if we're able to move this forward or whether you have other suggested edits as discussed.

**Comment by brijos** [comment]
Hey Jonah, happy to chat. First of all sorry for the late feedback. I reviewed the proposed blog late last week and I don't see it diving deep to qualify as a technical blog. The blog reads like the intro to your OpenSearch talk instead of being a technical blog post. Please add more substance such as interesting ways to correlate data between Prometheus metrics and Jaeger traces without compromising what will be an awesome talk next week at OpenSearch Con.

**Comment by jkowall** [comment]
I appreciate your feedback on the blog post. Unfortunately, given the current timeline, the fact that it needs to be published this week, I'm unable to incorporate additional technical details. I plan to share the post on LinkedIn as it stands if it's not going to be posted on the OpenSearch blog.

I understand the importance of timely feedback, and I hope we can streamline the review process for future content to ensure it aligns with expectations and deadlines. This was very frustrating versus other places where I contribute content. 

I'll agree to disagree as we get a lot of questions at Kubecon about the use of different observability tools to handle different observability signals between Jaeger, Prometheus, and OpenSearch. If you do not want to publish things which answer the questions for the typical user then I don't understand what is relevant for the blog.

**Pull request closed without merging**